### ISSUE TITLE
一些在CRNN上的问题

### ISSUE THREAD

**Issue by zuoxiang95** [wb-finalking/tensorflow#1]
Hello @wb-finalking ，您好：
    最近在看"Attentive Fashion Grammar Network for Fashion Landmark Detection and Clothing Category Classification"这篇论文，发现您有实现这篇论文，我看了一下代码，发现您只实现了category分类这一部分，有关于attribute分类这一部分没有实现。我想请问一下如果要多加一个全连接对attribute进行分类，它的loss用哪个比较好。因为attribute是mutil-label，感觉用softmax不太合适吧？

**Comment by wb-finalking** [comment]
It depends on whether your attributes are related. If they are independent, you can add a fully connected layer, whose  number of output is the same as your attributes. And then activate it through a sigmoid layer. The loss can be achieved with binary cross entropy; if there is a mutual exclusion relationship, using softmax instead of sigmoid. Thanks for your interest in my code, hope it can help you.
